mlflow/entities/trace_data.py

```diff
@@ -0,0 +1,19 @@
+from dataclasses import asdict, dataclass, field
+from typing import Any, Dict, List
+
+from mlflow.entities._mlflow_object import _MLflowObject
+from mlflow.entities.span import Span
+
+
+@dataclass
+class TraceData(_MLflowObject):
```

Comment:
Do we need to inherit `_MLflowObject`?

Reply:
`_MLflowObject` looks like an abstract class and its subclass needs to implement `from_proto`?

Reply:
Ah good catch! We shouldn't inherit for TraceData now, as it's not ready for proto conversion.